README.md

```diff
@@ -236,6 +236,8 @@ Note: Templates for which there is no export profile id listed in the config fil
 
 ### Naming the exported files
 
+Note that the Audit Title field has been deprecated by SafetyCulture in favour of the Audit Title Rule. However, setting Audit Title as the export filename will still work for audits using the new Audit Title Rule functionality.
```

Comment:
Note that when automatic Audit Title rules are set on the template, the Audit will not contain an Audit Title field by default. Regardless, the export filename setting will still work as expected using the automatically generated Audit name.